packages/ffe-icons/bin/import.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+
+const optimizer = require('./optimizer');
+const autoCrop = require('svg-autocrop');
+
+const importPath = path.join(__dirname, '../import');
+const destPath = path.join(__dirname, '../icons');
+
+const crop = true;
+
+const log = console.log;
+
+const normalizeName = fileName => {
+    let name = fileName.normalize('NFC');
+
+    name = name.toLowerCase();
+    name = name.replace('-ikon.svg', '');
+    name = name.replace('.svg', '');
+
+    name = name.replace('æ', 'ae');
+    name = name.replace('ø', 'o');
+    name = name.replace('å', 'aa');
+
+    //name = name.replace(/^[a-zA-Z0-9]/, '');
+
+    return `draft-${name}-ikon.svg`;
+};
+
+const main = async () => {
+    const importFolderExists = fs.existsSync(importPath);
+    const destPathExists = fs.existsSync(destPath);
+    if (!importFolderExists) {
+        throw new Error(`Import folder ($(importPath)) not found`);
+    }
+
+    if (!destPathExists) {
+        fs.mkdirSync(destPath);
+    }
+
+    const files = await fs.readdirSync(importPath, { encoding: 'utf-8' });
```

Comment:
I don't think there's much point `await`ing a synchronous function, it's just get transpiled down to an immediately resolved promise.